scripts/linters/js_ts_linter.py

```diff
@@ -74,7 +86,8 @@ def _parse_js_or_ts_file(filepath, file_content, **kwargs):
 
 
 def _get_expression_from_node_if_one_exists(
-        parsed_node, components_to_check):
+    parsed_node: esprima.nodes.Node, components_to_check: List[str]
```

Comment:
Seems like we're passing component names rather than components, so maybe call the last arg `possible_component_names` instead? (Make sure to update the docstring, too.)

Reply:
> maybe call the last arg possible_component_names instead

Done! 

> Make sure to update the docstring, too

Done!